app/src/main/java/com/example/deejai/RoomSelection/SelectRoomActivity.kt

```diff
@@ -5,7 +5,6 @@ import com.example.deejai.Constants.LOCALIZATION_CODE
 import android.location.Location
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
-import android.util.Log
```

Comment:
Even though it's good practice to remove unused imports, please address them in the right commit.